models/db/name.go

```diff
@@ -89,3 +88,8 @@ func IsUsableName(names, patterns []string, name string) error {
 
 	return nil
 }
+
+// IsValidUsername checks if name is valid
+func IsValidUsername(name string) bool {
```

Comment:
How about moving this function to user_model ?

Reply:
done